src/components/exchange/ConnectDevice.vue

```diff
@@ -131,14 +131,17 @@ export default class ConnectDevice extends Vue {
   }
 
   beforeMount() {
-    if (this.walletName === 'Liquality' && !this.showDialog) {
-      this.tryConnectLiquality();
+    if (this.peginTxState.bitcoinWallet === constants.WALLET_LIQUALITY && !this.showDialog) {
+      this.tryConnect();
+    } else if (this.peginTxState.bitcoinWallet === constants.WALLET_LEDGER && !this.showDialog) {
+      this.tryConnect();
```

Comment:
Seems like you could simplify the if clause
```suggestion
    if ((this.peginTxState.bitcoinWallet === constants.WALLET_LIQUALITY ||
          this.peginTxState.bitcoinWallet === constants.WALLET_LEDGER)
    && !this.showDialog) {
      this.tryConnect();
```